pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml

```diff
@@ -235,6 +235,26 @@ spec:
           workspace: release-workspace
       runAfter:
         - check-data-keys
+    - name: update-cr-status
```

Comment:
This won't really do anything, because none of the tasks in the pipeline is currently saving any results. See for example this pipeline: https://github.com/konflux-ci/release-service-catalog/blob/development/pipelines/rh-advisories/rh-advisories.yaml

You will see multiple tasks with parameter `resultsDirPath`. And then you can check one of the tasks to see how results are saved, e.g. here: https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/publish-pyxis-repository/publish-pyxis-repository.yaml (search for "result"). You would need to add something like that to the push-disk-images task.

Reply:
isn't it this one? https://github.com/konflux-ci/release-service-catalog/blob/fe8d9486c6d27716427c794c19e77b0270ea8ab5/tasks/collect-data/collect-data.yaml#L100

the `collect-data` task is included in the `push-disk-images` pipeline: https://github.com/konflux-ci/release-service-catalog/blob/fe8d9486c6d27716427c794c19e77b0270ea8ab5/pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml#L84

Reply:
`collect-data` just creates the empty dir for the results. And then the other tasks store the results.

Reply:
i've changed the input to be the snapshot, does it make sense?
https://github.com/konflux-ci/release-service-catalog/pull/717/commits/f4876993716823f1a2442ffa225067f3a556273e

Reply:
No, it does not. As I said, you will need to modify the push-disk-image task to save results similar to how other tasks save the results.

Reply:
the `push-disk-image` task gets the snapshot spec as input and acts on it. i want to publish that same input.
https://github.com/konflux-ci/release-service-catalog/blob/fe8d9486c6d27716427c794c19e77b0270ea8ab5/pipelines/push-disk-images-to-cdn/push-disk-images-to-cdn.yaml#L229-L230

Reply:
how is it looking now @mmalina? :pray: